### PR TITLE
Translate README to Finnish

### PR DESCRIPTION
**Pull request by Hassunaama**

### CODE REVIEW

File: README-fi.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# 6Dash
+API ja optimointi modi [3Dash](https://delugedrop.itch.io/3dash):ille
```
Comment (cgytrus) [inline README-fi.md:2]: is `:ille` supposed to be here?

File: README-fi.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+[English](README.md)
+
+## Ominaisuudet
+- Gizmos ([popcron/gizmos](https://github.com/popcron/gizmos) kautta)
+- Kenttä lautaus optimointi
+- Kääntyvät palikat
+- Muita ominaisuuksia jotka voivat auttaa 3Dash modaajia
+
+## Asennus
+1. Lataa BepInEx asennusohjelma [script](https://gist.githubusercontent.com/cgytrus/29085a6bf179893666316a36e1c92bf6/raw/bepinex-installer.ps1)
```
Comment (cgytrus) [inline README-fi.md]: `script` isn't translated?

File: README-fi.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+2. Laita asennus-scripti sinun 3Dash kansioon
+3. Käynnistä scripti oikea-klikkaamalla ja valitsemalla: `Run with Powershell`
```
Comment (cgytrus) [inline README-fi.md]: `Run with Powershell` isn't translated (is it not translated on Windows in Finnish either?)

**Review by cgytrus — CHANGES_REQUESTED** [review]

File: README-fi.md (hunk 1 above)
Comment (Hassunaama) [reply README-fi.md:2]: Yes, you can delete the : but it makes it more clear

File: README-fi.md (hunk 3 above)
Comment (Hassunaama) [reply README-fi.md]: It is not translated on Windows

File: README-fi.md (hunk 2 above)
Comment (Hassunaama) [reply README-fi.md]: It's really hard to translate it, should be fixed now.

File: README-fi.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ API ja optimointi modi [3Dash](https://delugedrop.itch.io/3dash):ille
 - Muita ominaisuuksia jotka voivat auttaa 3Dash modaajia
 
 ## Asennus
-1. Lataa BepInEx asennusohjelma [script](https://gist.githubusercontent.com/cgytrus/29085a6bf179893666316a36e1c92bf6/raw/bepinex-installer.ps1)
+1. Lataa BepInEx asennusohjelma. [Scripti](https://gist.githubusercontent.com/cgytrus/29085a6bf179893666316a36e1c92bf6/raw/bepinex-installer.ps1)
```
Comment (cgytrus) [inline README-fi.md]: i don't think the sentence is supposed to end here
also not translated on line 16 :p
you said it's hard to translate so if you think people will understand it, it's better to leave it in english

Comment (Hassunaama) [reply README-fi.md]: It should be fully translated, I tried to make it as clear as possible to read.

**Review by cgytrus — APPROVED** [review]

**Pull request merged**